docs/commandline/pouch_run.md

```diff
@@ -0,0 +1,55 @@
+## pouch run
+
+Create a new container and start it
+
+### Synopsis
+
+Create a container object in Pouchd, and start the container. This is useful when you just want to use one command to start a container. 
+
+```
+pouch run a container [flags]
+```
+
+### Examples
+
+```
+$ pouch run --name test registry.hub.docker.com/library/busybox:latest echo "hi"
+23f8529fddf7c8bbea70e2c12353e47dbfa5eacda9d58ff8665269614456424b
+$ pouch ps
+Name   ID       Status    Image                                            Runtime   Created
+test   23f852   stopped   registry.hub.docker.com/library/busybox:latest   runc      4 seconds ago
+pouch run -i --name test registry.hub.docker.com/library/busybox:latest echo "hi"
```

Comment:
I am afraid here we need a `$ ` at the beginning of the line. @HusterWan

Reply:
Ok, i will fix soon